File: Assignmentno8/task_5.c
#include<stdio.h>
void Display(int iNum)
{
    int iCnt = 0;
    for(iCnt = 10;iCnt>=1;iCnt--)
    {
        printf("%d\t",iCnt*iNum);
    }
}

int main()
{
    int iNo = 0;

    printf("Enter the number\n");
    scanf("%d",&iNo);

    Display(iNo);

    return 0;
}

File: Assignmentno16/task_5.java
import java.util.*;
class Arithematic
{
    void CountNumbers(int iNum)
    {
        int iCnt = 0;
        int iCount= 0;
        for(iCnt =1;iCnt<=iNum;iCnt++)
        {      
          iCount++;
          iNum = iNum/10;
        }
        System.out.println("Count is:"+iCount);   
    }
}


class task_5
{
    public static void main(String A[])
    {
        int iNo=0;
        Arithematic aobj = new Arithematic();
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the numbers:");
        iNo = sobj.nextInt();
        aobj.CountNumbers(iNo);


    }
}